packages/web-components/src/components/va-alert/va-alert.css

```diff
@@ -85,7 +85,7 @@ div.body {
 }
 
 .bg-only ::slotted(*) {
-  margin-top: 0;
+  margin-top: 0!important; /* Fix for Safari 15.3 and lower. */
```

Comment:
Using `!important` might be heavy-handed but I couldn't get this specificity to calculate higher than the styles that are getting favored in Safari without doing this. 

```
::slotted(:not([slot])) {
  margin-top: 2rem;
}
```